crates/iceberg/src/lib.rs

```diff
@@ -27,4 +28,5 @@ pub use error::Error;
 pub use error::ErrorKind;
 pub use error::Result;
 
+mod avro;
```

Comment:
Do we need to expose some API from this mod?

Reply:
Currently no.